### PR TITLE
KFSPTS-27905 fix PREQ exception

### PR DESCRIPTION
**Pull request by db483**

### CODE REVIEW

File: src/main/resources/edu/cornell/kfs/module/purap/document/validation/configuration/PurapValidatorDefinitions.xml [hunk 1]
```diff
@@ -125,6 +125,10 @@
 	<bean id="VendorCreditMemo-wireTransferValidation" class="edu.cornell.kfs.module.purap.document.validation.impl.CreditMemoWireTransferValidation" abstract="true" />
 	<bean id="VendorCreditMemo-foreignDraftValidation" class="edu.cornell.kfs.module.purap.document.validation.impl.CreditMemoForeignDraftValidation" abstract="true" />
 
+    <bean id="PaymentRequest-accountingLineAccessibleValidation"
+          class="org.kuali.kfs.module.purap.document.validation.impl.PaymentRequestAccountingLineAccessibleValidation"
+          abstract="true" p:dataDictionaryService-ref="dataDictionaryService" p:documentDictionaryService-ref="documentDictionaryService"/>
```
Comment (jhulslander) [inline src/main/resources/edu/cornell/kfs/module/purap/document/validation/configuration/PurapValidatorDefinitions.xml:130]: THe kualico version of this overlay doesn't include documentDictionaryService-ref, can you confirm we want that

**Review by jhulslander — COMMENTED** [review]
Change looks good, I just want to confirm one thing

File: src/main/resources/edu/cornell/kfs/module/purap/document/validation/configuration/PurapValidatorDefinitions.xml (hunk 1 above)
Comment (jhulslander) [reply src/main/resources/edu/cornell/kfs/module/purap/document/validation/configuration/PurapValidatorDefinitions.xml:130]: ah, never mind, this isn't an extension, I shall merge

Comment (db483) [reply src/main/resources/edu/cornell/kfs/module/purap/document/validation/configuration/PurapValidatorDefinitions.xml:130]: yes, that is the missing part that I have added

**Review by jhulslander — APPROVED** [review]
change look good, I shall merge

**Pull request merged**